### PR TITLE
Extract algorithm consts and helper into utilities

### PR DESCRIPTION
**Pull request by kaifeng**
This should be something in the common but there is already
a common file thus renamed to utils. Other common stuff can
be put to the utils as needed.

### CODE REVIEW

**Review by mpeters — APPROVED** [review]
lgtm

File: keylime/utils/algorithms.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+"""
+SPDX-License-Identifier: Apache-2.0
+Copyright 2017 Massachusetts Institute of Technology.
```
Comment (lukehinds) [inline keylime/utils/algorithms.py:3]: @kaifeng you can put you're own name there is you like.  May as well make it 2020 as well (as its a new file).

Comment (kaifeng) [reply keylime/utils/algorithms.py]: okay, just haven't seen many personal copyright in opensource projects.

**Comment by mpeters** [comment]
@kaifeng do you want to merge this yourself? I think in general we are going to let the committer do the merge (if they have permissions) as this gives them the most flexibility for timing, etc depending on what other changes they have in the pipeline.

**Comment by kaifeng** [comment]
@mpeters well, it seems I can't merge myself.

**Pull request merged**